apps/src/templates/projects/projectsRedux.js

```diff
@@ -1,4 +1,7 @@
 /** @file Redux actions and reducer for the Projects Gallery */
+
+/* eslint-disable import/order */
+// Disabling import order due to circular dependencies.
```

Comment:
same as above; I'd love to scope this disable directive down to just the imports that need it, and document more details about what exactly goes wrong when we don't disable.